$OpenBSD: patch-configure,v 1.17 2019/03/13 06:57:14 ajacoutot Exp $

XXX
Package >= was not found in the pkg-config search path
Package fribidi_required_dep was not found in the pkg-config search path

Index: configure
--- configure.orig
+++ configure
@@ -27249,7 +27249,7 @@ $as_echo "yes" >&6; }
 fi
 
 GTK_PACKAGES="atk >= 2.15.1 cairo >= 1.14.0 cairo-gobject >= 1.14.0 gdk-pixbuf-2.0 >= 2.30.0 gio-2.0 >= 2.53.4"
-GTK_PRIVATE_PACKAGES="$ATK_PACKAGES $WAYLAND_PACKAGES $MIR_PACKAGES epoxy >= 1.4 fribidi >= fribidi_required_dep"
+GTK_PRIVATE_PACKAGES="$ATK_PACKAGES $WAYLAND_PACKAGES $MIR_PACKAGES epoxy >= 1.4 fribidi"
 if test "x$enable_x11_backend" = xyes -o "x$enable_wayland_backend" = xyes; then
   GTK_PRIVATE_PACKAGES="$GTK_PRIVATE_PACKAGES pangoft2"
 fi
